app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -288,10 +340,20 @@ def add_instance_disk(disks, size, name, location)
         super(disks, size, location, name, "azure")
       end
 
+      def get_security_groups(hardware_network_info)
+        security_group_array = hardware_network_info.collect { |info| info.delete(:security_group) }.compact
+        hardware_network_info.delete_if { |e| e.kind_of?(Hash) && e.empty? }
+
+        process_collection(security_group_array, :security_groups) do |sg|
+          parse_security_group(sg)
+        end
+      end
+
       def get_hardware_network_info(instance)
         networks_array = []
 
         get_vm_nics(instance).each do |nic_profile|
+          networks_array << {:security_group => get_nic_security_group(nic_profile)}
```

Comment:
@djberg96 -Can you remind me why we set the security_groups property on the network object? Is there a security_groups column for hardware.networks?